src/store/posts.js

```diff
@@ -0,0 +1,42 @@
+import api from 'axios'
+
+const state = {
+  posts: []
+}
+
+const getters = {
+  postsList: state => {
+    return state.posts
+  }
+}
+
+const mutations = {
+  setPosts (state, value) {
+    state.posts = value
+  },
+  addPosts (state, value) {
```

Comment:
mutation está sendo utilizada?

Reply:
por enquanto não

Reply:
então não seria melhor remove-la?

Reply:
addPosts foi retirado

Reply:
a mutation não está sendo utilizada? ou ela será utilizada nesse processo mas ainda não foi chamada? será feito nesse momento ou talvez futuramente? só pra entender a necessidade de retirar ou não, porque se for algo que está sendo construído pra agora não precisa retirar.

Reply:
ainda não está sendo utilizada mesmo, caso haja necessidade de utilizar ela eu coloco de volta